### PR TITLE
Open-Pull

### PR DESCRIPTION
**Pull request by Nguyenpham89**
- [ ] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----nguyenpham89
F02F3F65-2796-4BC9-86AB-623E7F39BCC2

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
   stale:
```
Comment (Usrwrr) [inline .github/workflows/stale.yml:8]: ۱۱

**Review by Usrwrr — COMMENTED** [review]
Srwrrjy

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ jobs:
...
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/stale@v4
+    - uses: actions/stale@v5
```
Comment (Usrwrr) [inline .github/workflows/stale.yml:11]: 11

**Review by Usrwrr — COMMENTED** [review]
@[6037701726651810](مقصودرحمتی)

File: .github/workflows/stale.yml (hunk 1 above)
Comment (Usrwrr) [reply .github/workflows/stale.yml:8]: ۴۵۹۶

**Review by Vailoa — COMMENTED** [review]
Aua ke fia kama

**Comment by ahpook** [comment]
Spam.

**Pull request closed without merging**